executor/point_get.go

```diff
@@ -65,6 +64,18 @@ type PointGetExecutor struct {
 	lock     bool
 }
 
+// Init set fields needed for PointGetExecutor reuse, this does NOT change baseExecutor field
+func (e *PointGetExecutor) Init(p *plannercore.PointGetPlan, startTs uint64) error {
```

Comment:
This function never returns error

Reply:
current implemntation could signature return value as nil, keep error return for possible  future usage?

Reply:
We can always add an error later when needed.